src/components/RepoCard/RepoCard.css

```diff
@@ -0,0 +1,49 @@
+.inner {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(550px, 1fr));
+  gap: 20px;
+}
+.item {
+  width: 550px;
+  height: 300px;
+  border: 1px solid #ddd;
+  border-radius: 8px;
+  padding: 30px 20px;
+  background-color: #fff;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+}
+
+.item__title {
+  font-size: 50px;
+  margin-bottom: 10px;
```

Comment:
Also make sure to read about the concept of spacing units.
In general UI system would have 
```ts
const spacingUnit = 8;

const spacing = (multiplier) => spacingUnit * multiplier;

marginBottom = spacing(1)
padding = spacing(2)
```

I've written in js but same concept applies to css. All spacing should be a factor of base spacing unit. Usually it's 8px.
So you end up with 4px, 8px, 16px, 32px etc. That way it looks nice. Read about it in chatpgt